packages/auth/src/grant/routes.ts

```diff
@@ -364,16 +364,16 @@ async function revokeGrant(
     'GNAP '
   )[1]
   if (!continueId || !continueToken) {
-    ctx.throw(401, { error: 'invalid_request' })
+    ctx.throw(401, { error: { code: 'invalid_request', description: 'invalid continuation information' } })
```

Comment:
Similar to how you did above, should we add the description also as the second parameter to have it also exposed to the caller/client (like postman)? Or not for these 401 errors?